clang/lib/Format/QualifierAlignmentFixer.h

```diff
@@ -1,5 +1,4 @@
-//===--- LeftRightQualifierAlignmentFixer.h ------------------------------*- C++
-//-*-===//
+//===---  QualifierAlignmentFixer.h ------------------------------*- C++-*-===//
```

Comment:
There should be a single space before `QualifierAlignmentFixer.h`.

Reply:
Ok, great catch and great opportunity to go over the work flow of comment and then correction. thank you Owen.

Reply:
Not entirely sure @owenca how to tell you that I've "Done" that comment... will miss the inline comments.

Reply:
Just use `Resolve conversation`. That should be available to the author and commenter. You won't get a notification about it.

The inline comments are available if you switch to the `Files changed` tab. They are visible if they belong to the version you commented on. Otherwise there's `Conversations` at the top which let's you see the outdated ones as well (just discovered that myself).